devel/moth.py

```diff
@@ -128,6 +163,14 @@ def read_stream(self, stream):
                     # Make sure this shows up in the header block of the HTML output.
                     self.files[val] = PuzzleFile(stream, val, visible=False)
                     self.scripts.append(val)
+                elif key == "objective":
+                    self.objective = val
+                elif key == "success":
+                    self.success = val
```

Comment:
I think for the rfc822 parser we're going to need to read success.acceptable and success.mastery separately, right?

Reply:
Added in 03d1182e73ffc740d8ac4decdf89c8194d445cf4